proposed/http-client/http-client.md

```diff
@@ -12,11 +12,12 @@ interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 ### Client
 
 An HTTP client has the responsibility to send a PSR-7 request and return a PSR-7
-response. Under the hood, the HTTP client MAY modify the request received from the user
-and/or the response received from the server. In this case, the request and the response
-MUST be consistent between the body and headers. For example, a server may return a gzip
-encoded body and the client may decides to decode the body. If the client decodes the body,
-the client MUST also remove the Content-Encoding header and adjust the Content-Length header.
+response. Under the hood, the HTTP client MAY modify the request received from
+the user and/or the response received from the server. In this case, the request
+and the response MUST be consistent between the body and headers. For example, a
+server may return a gzip-encoded body and the client may decide to decode the
+body. If the client decodes the body, the client MUST also remove the
+`Content-Encoding` header and adjust the `Content-Length` header.
```

Comment:
The main changes here are hyphenating "gzip-encoded`, and adding code tags around the header types.